pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -471,6 +471,9 @@ type QuotaSpec struct {
 	// MaxBytes represents the quota in bytes
```

Comment:
How about a comment here about preferring to use MaxSize instead?

Reply:
change made